package.json

```diff
@@ -1,15 +1,21 @@
 {
-  "name": "access-log-counter",
-  "version": "1.0.0",
-  "description": "",
-  "main": "count-traffic.js",
-  "dependencies": {
-    "n-readlines": "^1.0.1"
```

Comment:
Switched to `event-stream` instead, so that it's possible to read huge log files without computer freezing.

Reply:
Nice job!